src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapper.java

```diff
@@ -13,7 +18,20 @@ public SupplementaryEvidenceMapper() {
         // empty mapper construct
     }
 
-    public SupplementaryEvidence mapEnvelope(Envelope envelope) {
-        return new SupplementaryEvidence(mapDocuments(envelope.documents));
+    public SupplementaryEvidence map(List<Document> existingDocs, List<Document> envelopeDocs) {
```

Comment:
I think we should either split the mapping and combining of documents or perhaps rename this class and the method.

Reply:
ok for separate PR?
perhaps `SupplementaryEvidenceBuilder`?

Reply:
shouldn't this be in `DocumentMapper` instead? judging from arguments

Reply:
it's creating an object of type `SupplementaryEvidence`